### DISCUSSION TITLE
How to convert flac to mp3 without deleting the existing tags?

### DISCUSSION THREAD

**Discussion by crathm** [enzo1982/freac#344] [Audio Format Discussion]
Hello,

does somebody know how to make it?

**Comment by enzo1982** [comment]
Which tags are missing after conversion?

With default settings, fre:ac tries to transfer tags during conversions. However, FLAC uses the Vorbis Comment tagging format, while MP3 uses ID3v2 tags, so tag fields need to be mapped from one format to the other and the target format might not support all the fields you had in your original tags. Also, fre:ac might not recognize some of the tags in your FLAC files.

So it would be helpful if you could tell which tag fields are missing exactly.

**Reply by Ehfblion** [reply]
Closest thread I found hopefully you are still replying. 

I'm converting from flac to AIFF and then importing into rekordbox, using SndFile Output Converterv1.2.0, I'm fairly certain that AIFF uses ID3v2 but mentioning it for full picture. The default setting UTF-16LE and UTF-16BE (would love to know the difference, but not the key issue) results in converted files missing artwork. 

UTF-8 output had artwork but in case of multiple Artists, only the last Artist was displayed, whom wasn't even the primary one. example: 

![Screenshot 2025-04-28 223149](https://github.com/user-attachments/assets/a63de066-695f-4b17-933d-50858809e317)

I tried using the ISO 8859-1 encoding to have the same result. 

The last thing I noticed, is that the files are already simplified (abridged) when they are added to the joblist:

![Screenshot 2025-04-28 223621](https://github.com/user-attachments/assets/218ed4f3-7c4a-4a11-b46d-f091ade45cba)

And flac metadata format does not support different encoding protocols. So at the moment my guess is that the metadata is abridged upon import prior to conversion. And my question is, How do I expand list of supported formats ? or let me know if I took a wrong turn anywhere.
